app/views/hq/orphans/_form.html.erb

```diff
@@ -0,0 +1,151 @@
+<%= form_for [:hq, @orphan] do |f| %>
+  <%= render 'shared/errors', object: f.object %>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">Orphan Details</h3>
+    </div>
+    <div class="panel-body">
+      <div class="form-group">
+        <%= f.label :name %>
+        <%= f.text_field :name, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :date_of_birth %>
+        <%= f.text_field :date_of_birth, class: "form-control date-picker", value: @orphan.date_of_birth %>
+      </div>
+      <div class="form-group">
+        <%= f.label :gender %>
+        <%= f.select :gender,
+          Settings.lookup.gender, {}, class: "form-control" %>
+      </div>
+      <%# Health status %>
```

Comment:
Have a look at the schema: https://github.com/AgileVentures/osra/blob/develop/db/schema.rb#L133